metricbeat/module/system/_meta/docs.asciidoc

```diff
@@ -19,3 +19,104 @@ directly.
 The System module comes with a predefined dashboard. For example:
 
 image::./images/metricbeat_system_dashboard.png[]
+
+[float]
+=== Required permissions
+
+The System metricsets collect different kinds of metric data, which may require dedicated permissions
+to be fetched. For security reasons it's advised to grant the lowest possible permissions. This section
+justifies which permissions must be present for particular metricsets.
+
+Please notice that modern Linux implementations divide the privileges traditionally associated with superuser
+into distinct units, known as capabilities, which can be independently enabled and disabled.
+Capabilities are a per-thread attribute.
+
+==== cpu
+
+CPU statistics (idle, irq, user, system, iowait, softirq, cores, nice, steal, total) should be available without
+elevated permissions.
+
+==== load
+
+CPU load data (1 min, 5 min, 15 min, cores) should be available without elevated permissions.
+
+==== memory
+
+Memory statistics (swap, total, used, free, actual) should be available without elevated permissions.
+
+==== network
+
+Network metrics for interfaces (in, out, errors, dropped, bytes, packets) should be available without escalated
```

Comment:
also use elevated here to be consistent?

Reply:
Thanks, fixed.